tests/dummy/app/index.html

```diff
@@ -6,20 +6,19 @@
     <title>Ember Engines</title>
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="shortcut icon" href="/images/favicon.png">
```

Comment:
we need keep this favicon and also find a place to use our brand image - https://avatars3.githubusercontent.com/u/23064483?s=200&v=4

Reply:
The favicon is currently in the project as public/images/favicon, I added this back.